examples/pixartalpha_example.py

```diff
@@ -1,22 +1,18 @@
 import time
+import os
 import torch
 import torch.distributed
 from xfuser import xFuserPixArtAlphaPipeline, xFuserArgs
 from xfuser.config import FlexibleArgumentParser
-from xfuser.distributed import (
-    get_world_group, 
-    get_data_parallel_rank, 
-    get_data_parallel_world_size,
-    get_runtime_state
-)
+from xfuser.distributed import ps, rs
```

Comment:
> I don't think it is a good idea to use representations like `ps` and `rs` as they add code with little benefit. It seems to me that it would be more straightforward and convenient to use the entire distributed directory for runtime environment management and import the required functions directly from xfuser.distributed

使用ps和rs后，就不用每次需要某个函数后，专门再去导入一下，可以节省一些不必要的导入，我看Megatron是用mpu，DeepSpeed是用groups来组织这些函数的。

Reply:
Maybe a statement like `import xfuser.distributed as (a name like mpu)` (or rename distributed dir) could be used to solve the problem? It doesn't feel like it's necessary to distinguish between ps and rs, respectively.

Reply:
我认为没有必要用ps和rs，并没有给使用带来便利，而且ps和rs这两个名称非常confusing

Reply:
> 我认为没有必要用ps和rs，并没有给使用带来便利，而且ps和rs这两个名称非常confusing

使用上的便利还是有的，第一就是需要用某个函数时，不需要显示导入了，第二就是编码时，更利于智能提示，比如在vscode里面，输入ps.,就会把候选函数列出来。这2个简写名称可能需要改一下？